### PR TITLE
Misc design tweaks that get style closer to mockups

### PR DESCRIPTION
**Pull request by i386**
@reviewbybees

### CODE REVIEW

File: less/theme.less [hunk 1]
```diff
@@ -26,10 +26,12 @@ html {
...
     color: @text-color;
-    text-rendering: optimizeSpeed;
-    font-feature-settings: "kern" on, "liga" off;
-    -moz-font-feature-settings: "kern" on, "liga" off;
-    //TODO: Do we really want this for everything? -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeSpeed;
+  font-feature-settings: "kern" on, "liga" off;
+  -moz-font-feature-settings: "kern" on, "liga" off;
+  // Want to change this? Pick a fight with i386
+  -webkit-font-smoothing: antialiased;
```
Comment (sophistifunk) [inline less/theme.less]: worse-than-hitler

File: less/theme.less [hunk 2]
```diff
@@ -642,6 +664,10 @@ code.hash {
     a {
         color: white;
     }
+
+    h1 {
+      font-size: 1.8rem;
```
Comment (sophistifunk) [inline less/theme.less]: Not `@font-size-h1`?

**Comment by sophistifunk** [comment]
LGTM 🐝

File: less/theme.less (hunk 1 above)
Comment (i386) [inline less/theme.less]: haha

File: less/theme.less (hunk 2 above)
Comment (i386) [inline less/theme.less]: Nah mate – the standard H1 is a bit too big to use in the nav.

**Pull request merged**